tutorials/misc/plot_ecog.py

```diff
@@ -11,192 +11,169 @@
+# the coordinate frame of the montage
+print(raw.get_montage().get_positions()['coord_frame'])
 
-# set channel types to ECoG (instead of EEG)
-raw.set_channel_types({ch_name: 'ecog' for ch_name in raw.ch_names})
+# Find the annotated events
+events, event_id = mne.events_from_annotations(raw)
 
-###############################################################################
-# We can then plot the locations of our electrodes on our subject's brain.
-# We'll use :func:`~mne.viz.snapshot_brain_montage` to save the plot as image
-# data (along with xy positions of each electrode in the image), so that later
-# we can plot frequency band power on top of it.
-#
-# .. note:: These are not real electrodes for this subject, so they
-#           do not align to the cortical surface perfectly.
+# Make a 25 second epoch that spans before and after the seizure onset
+epoch_length = 25  # seconds
+epochs = mne.Epochs(raw, events, event_id=event_id['onset'],
+                    tmin=13, tmax=13 + epoch_length, baseline=None)
 
-fig = plot_alignment(raw.info, subject=subject, subjects_dir=subjects_dir,
-                     surfaces=['pial'], trans=trans, coord_frame='mri')
-mne.viz.set_3d_view(fig, 200, 70, focalpoint=[0, -0.005, 0.03])
+# And then load data and downsample.
+# .. note: This is just to save execution time in this example, you should
+#          not need to do this in general!
+epochs.load_data()
+epochs.resample(200)  # Hz, will also load the data for us
 
-xy, im = snapshot_brain_montage(fig, montage)
+# Finally, make evoked from the one epoch
+evoked = epochs.average()
 
-###############################################################################
-# Next, we'll compute the signal power in the gamma (30-90 Hz) and alpha
-# (8-12 Hz) bands.
-gamma_power_t = raw.copy().filter(30, 90).apply_hilbert(
-    envelope=True).get_data()
-alpha_power_t = raw.copy().filter(8, 12).apply_hilbert(
-    envelope=True).get_data()
-gamma_power = gamma_power_t.mean(axis=-1)
-alpha_power = alpha_power_t.mean(axis=-1)
 
 ###############################################################################
-# Now let's use matplotlib to overplot frequency band power onto the electrodes
-# which can be plotted on top of the brain from
-# :func:`~mne.viz.snapshot_brain_montage`.
+# Explore the electrodes on a template brain
+# ------------------------------------------
+#
+# Our electrodes are shown after being morphed to fsaverage brain so we'll use
+# this fsaverage brain to plot the locations of our electrodes. We'll use
+# :func:`~mne.viz.snapshot_brain_montage` to save the plot as image data
+# (along with xy positions of each electrode in the image), so that later
+# we can plot frequency band power on top of it.
 
-# Convert from a dictionary to array to plot
-xy_pts = np.vstack([xy[ch] for ch in raw.info['ch_names']])
+fig = plot_alignment(raw.info, subject='fsaverage', subjects_dir=subjects_dir,
+                     surfaces=['pial'], coord_frame='mri')
```

Comment:
```suggestion
                     surfaces=['pial'], coord_frame='mni_tal')
```

Reply:
This causes an error... you can't pass that into `coord_frame`, it's already in `fsaverage` because we're using `subject = 'fsaverage'`

Reply:
Ah I see you're right.